src/sections/section1/kopf.svg

```diff
@@ -0,0 +1,115 @@
+				 M271.624,91.72l25.62,25.617c-3.983,3.009-6.998,7.118-8.672,11.82H268.36c-2.015-5.65-5.953-10.411-11.125-13.451
+				L271.624,91.72z M218.581,76.45l27.792-5.558c2.181,5.03,5.917,9.23,10.658,11.983l-15.49,25.817L214.763,91.51
+				c2.58-4.051,3.956-8.751,3.97-13.553C218.733,77.441,218.61,76.958,218.581,76.45z M252.866,137.691
+				c0,4.713-3.82,8.533-8.533,8.533c-4.711-0.005-8.529-3.822-8.533-8.533c0-4.713,3.82-8.533,8.533-8.533
+				S252.866,132.978,252.866,137.691z M193.133,69.424c4.713,0,8.533,3.82,8.533,8.533s-3.82,8.533-8.533,8.533
+				s-8.533-3.82-8.533-8.533C184.604,73.246,188.422,69.429,193.133,69.424z M183.845,101.75c5.199,2.144,10.987,2.389,16.349,0.693
+				l26.347,16.905c-2.804,2.747-4.938,6.104-6.235,9.809h-37.279c-1.863-5.282-5.405-9.809-10.085-12.886L183.845,101.75z
+				 M158.999,146.224c-4.713,0-8.533-3.82-8.533-8.533c0.005-4.711,3.822-8.529,8.533-8.533c4.713,0,8.533,3.82,8.533,8.533
+				S163.712,146.224,158.999,146.224z M184.599,223.024c-4.713,0-8.533-3.821-8.533-8.533c0-4.713,3.82-8.533,8.533-8.533
+				s8.533,3.821,8.533,8.533C193.128,219.202,189.31,223.019,184.599,223.024z M189.452,189.381
+				c-1.598-0.318-3.224-0.482-4.853-0.491c-0.845,0-1.643,0.169-2.467,0.25l-11.628-28.707c5.839-2.959,10.32-8.045,12.522-14.209
+				h37.279c1.537,4.298,4.198,8.105,7.706,11.024L189.452,189.381z M254.439,205.957h-45.813c-0.911-2.556-2.226-4.95-3.894-7.09
+				l41.909-34.924l14.271,32.008C257.996,198.729,255.778,202.158,254.439,205.957z M268.36,146.224h20.213
+				c2.015,5.65,5.953,10.411,11.125,13.451l-17.742,29.569c-1.154-0.198-2.319-0.316-3.489-0.353c-0.661,0-1.282,0.146-1.931,0.195
+				l-14.649-32.855C264.803,153.452,267.021,150.023,268.36,146.224z M278.466,223.024c-4.713,0-8.533-3.821-8.533-8.533
+				c0-4.713,3.821-8.533,8.533-8.533c4.713,0,8.533,3.821,8.533,8.533C286.995,219.202,283.177,223.019,278.466,223.024z
+				 M314.891,167.52l23.831,19.065c-3.749,3.2-6.49,7.419-7.89,12.145l-28.806,5.761c-1.187-2.727-2.839-5.226-4.883-7.386
+				L314.891,167.52z M329.666,308.357c-4.713,0-8.533-3.82-8.533-8.533s3.82-8.533,8.533-8.533s8.533,3.82,8.533,8.533
+				C338.195,304.535,334.377,308.353,329.666,308.357z M363.799,265.691c0.031,2.851,0.544,5.676,1.517,8.356l-14.405,11.515
+				c-5.386-8.413-15.323-12.737-25.15-10.943l-28.329-43.091c2.531-2.822,4.399-6.173,5.468-9.809l28.806-5.761
+				c5.492,12.971,20.439,19.057,33.429,13.611l10.527,14.592C368.288,248.839,363.814,256.958,363.799,265.691z M372.11,186.831
+				l16.282-40.709c0.341,0.014,0.662,0.102,1.007,0.102c1.028-0.04,2.053-0.144,3.069-0.31l10.332,20.611
+				c-8.226,4.438-13.368,13.019-13.401,22.366c0,0.346,0.089,0.667,0.102,1.01l-11.473,4.587
+				C376.561,191.583,374.552,188.984,372.11,186.831z M389.399,274.224c-4.713,0-8.533-3.82-8.533-8.533
+				c0.005-4.711,3.822-8.529,8.533-8.533c4.713,0,8.533,3.82,8.533,8.533S394.112,274.224,389.399,274.224z"/>
+		</g>
+	</g>
+</g>
+<g>

```

Comment:
Nils hat vorgeschlagen, dass man die überflüssigen <g>'s löscht - what do you think?